pkg/baseline/packageIndex.json

```diff
@@ -9,8 +9,9 @@
       "BaselineVersion": "4.4.0",
```

Comment:
I see. I think we should change this to 4.5.0 in the case as we want all others (products and tests) to reference the latest package we are newly building. Thanks @ericstj.